state/kafka-sender/src/test/java/com/expediagroup/streamplatform/streamregistry/state/kafka/KafkaEventReceiverTest.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.state.kafka;
+
+
+
+import lombok.extern.slf4j.Slf4j;
+
+
+
+
+@Slf4j
+public class KafkaEventReceiverTest {
+
+//  @Rule
```

Comment:
Can we either enable or remove this test.

Reply:
Sorry, accidentally staged. Will get cleaned up when I pushed the tests.

Reply:
done